ui/components/Lifecycle/Environments/index.js

```diff
@@ -598,13 +590,13 @@ const Environments = ({ organization, classes }) => {
                   originalLeftCount={connections?.total_count}
                   originalRightCount={environmentConnections?.total_count}
                   leftPermission={CAN(
-                    keys.REMOVE_CONNECTIONS_FROM_ENVIRONMENT.action,
-                    keys.REMOVE_CONNECTIONS_FROM_ENVIRONMENT.subject,
-                  )}
-                  rightPermission={CAN(
                     keys.ASSIGN_CONNECTIONS_TO_ENVIRONMENT.action,
                     keys.ASSIGN_CONNECTIONS_TO_ENVIRONMENT.subject,
                   )}
+                  rightPermission={CAN(
```

Comment:
Why reverse keys?
The left is of remove and right of assign.

Reply:
please check the other areas

Reply:
@Yashsharma1911 help! 😜

Reply:
@sudhanshutech I just checked.
It seems that Akshay made a mistake while making his migration to transferList components at other places.
As I stated, and as you have defined in sistent, the left permission is used to move components to left side, which is the remove action.